docker-compose.yml

```diff
@@ -0,0 +1,39 @@
+version: '2'
+services:
+    mysql.local:
+        image: keybaseprivate/mysql
+        restart: unless-stopped
```

Comment:
Yeah I think it'd be a good idea to remove this and the copy lower in the file. should make things more predictable.